packages/web/app/src/lib/preflight/graphiql-plugin.tsx

```
@@ -170,13 +169,13 @@ export function usePreflightScript(args: {
   const [state, setState] = useState<PreflightWorkerState>(PreflightWorkerState.ready);
   const [logs, setLogs] = useState<LogRecord[]>([]);
 
-  const currentRun = useRef<null | Function>(null);
+  const currentExecution = useRef<null | Function>(null);
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)